README.md

```diff
@@ -0,0 +1,32 @@
+OregonDigital\_2
+===============
+
+[![Circle CI](https://circleci.com/gh/OregonDigital/oregondigital_2.svg?style=svg)](https://circleci.com/gh/OregonDigital/oregondigital\_2)
+
+
+Local Development Setup
+-----
+
+Requirements:
+
+  **Ruby Version: 2.0**
+
+Configuration and database creation:
+
+    git clone git@github.com:OregonDigital/oregondigital_2.git
+	cd oregondigital_2
+	bundle install
+	rake db:migrate
+	rake jetty:clean
+	rake jetty:config
+
+Run tests with RSpec:
```

Comment:
This might make sense to put after the jetty:start command below, as rspec just pukes out failures if jetty isn't running